include/IDescriptorSet.h

```diff
@@ -69,7 +69,7 @@ class IDescriptorSet : public virtual IVulkanPrebaked
                         texture->drop();
                 }
 
-                IVirtualTexture* texture;
+                IRenderableVirtualTexture* texture;
```

Comment:
if you're gonna grab and drop things, use smart ptr

Reply:
@Crisspl ?

Reply:
This is file created by me, I think it needs to die

there might be similar stubs around the `include` directory